processor/src/test/kotlin/io/mcarle/konvert/processor/konvert/KonvertITest.kt

```diff
@@ -1688,6 +1688,62 @@ interface DtoMapper {
 //            dtoMapperCode
 //        )
     }
+
+    @Test
+    fun suspendFun() {
+        val (compilation) = compileWith(
+            enabledConverters = listOf(SameTypeConverter()),
+            code = SourceFile.kotlin(
+                name = "TestCode.kt",
+                contents =
+                """
+import io.mcarle.konvert.api.Konverter
+
+class SourceClass(val property: SourceProperty)
+class TargetClass(val property: TargetProperty)
+
+class SourceProperty(val value: String)
+class TargetProperty(val value: String)
+
+@Konverter
+interface Mapper {
+    suspend fun toTarget(source: SourceClass): TargetClass
+    suspend fun toTarget(source: SourceProperty): TargetProperty =
+        TargetProperty(source.value)
+}
+                """.trimIndent()
+            )
+        )
+        val mapperCode = compilation.generatedSourceFor("MapperKonverter.kt")
+        println(mapperCode)
+
+        assertContains(mapperCode, "suspend fun toTarget(source: SourceClass): TargetClass")
+        assertContains(mapperCode, "suspend fun toTarget(source: SourceProperty): TargetProperty")
+    }
+
+    @Test
+    fun internalInterface() {
+        val (compilation) = compileWith(
+            enabledConverters = listOf(SameTypeConverter()),
+            code = SourceFile.kotlin(
+                name = "TestCode.kt",
+                contents =
+                """
+import io.mcarle.konvert.api.Konverter
+
+class SourceClass(val property: String)
+class TargetClass(val property: String)
+
+@Konverter
+internal interface Mapper {
+    fun toTarget(source: SourceClass): TargetClass
+}
+                """.trimIndent()
+            )
+        )
+        val mapperCode = compilation.generatedSourceFor("MapperKonverter.kt")
+        println(mapperCode)
+    }
```

Comment:
Please add an assertion here (like you have done for `suspendFun`) to verify the internal keyword is present in the mapper code.

Reply:
Ouch! My bad.